src/khoj/configure.py

```diff
@@ -229,11 +230,16 @@ def configure_server(
 
         state.SearchType = configure_search_types()
         state.search_models = configure_search(state.search_models, state.config.search_type)
+        setup_default_agent()
         initialize_content(regenerate, search_type, init, user)
     except Exception as e:
         raise e
 
 
+def setup_default_agent():
+    AgentAdapters.create_default_agent()
```

Comment:
We should only `create_default_agent` if it doesn't already exist. Creating one everytime app starts seems unnecessary

Reply:
This is so that it can easily keep in sync as the system prompt changes. In the method, it only creates an actual _new_ agent if it doesn't exist. Otherwise, it just updates the fields.